components/controlnet_aux_processor/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM --platform=linux/amd64 pytorch/pytorch:2.0.1-cuda11.7-cudnn8-devel
```

Comment:
I think you can change the base image to the `runtime` one instead of the `devel` one since it's more lightweight and should do the job -> `pytorch/pytorch:2.0.1-cuda11.7-cudnn8-runtime`

I don't think you need all the additional stuff that come with the `devel` version. 

https://stackoverflow.com/questions/56405159/what-is-the-difference-between-devel-and-runtime-tag-for-a-docker-container

Reply:
> Nice one! :) Thanks for adding.
> 
> Left a few comments but should be good to go

I will still wait for the PR on the controlnet_aux repo to be merged, it's still in review :D Then I can test/merge it!